lib/include.js

```diff
@@ -47,6 +47,8 @@ function normalizeInclude(include) {
       newInclude = newInclude.concat(subIncludes);
     }
     return newInclude;
+  } else if (typeof include === 'boolean' || typeof include === 'number') {
+    return [];
   } else {
     return include;
```

Comment:
It appears that the normalizeInclude function is expected to transform the original value into an array, and this still does not handle cases for "undefined", "symbol", "function", null or other special objects if I understand correctly? Should probably add some additional test cases to account for other data types to ensure that the behaviour is correct. Perhaps someone with some more knowledge on this code can weigh in.

Reply:
+1 to make these checks more generic and ignore all invalid `include` values.

To make this situation easier to troubleshoot, we should at least print a `debug` statement to let the callers know about the problem.

It would be even better to reject such queries with an error, but I understand that may be too much work.